aten/src/ATen/cpu/vec/vec256/vec256_convert.h

```diff
@@ -297,20 +297,35 @@ struct VecConvert<
     return convert_int8_to_float<src_t>(src[0]);
   }
 };
+#elif defined(CPU_CAPABILITY_NEON)
+template <typename src_t>
+struct VecConvert<
+    float,
+    2,
+    src_t,
+    1,
+    typename std::enable_if_t<is_8bit_integer_v<src_t>,
+        void>> {
+  static inline VectorizedN<float, 2> apply(const VectorizedN<src_t, 1>& src) {
+    const auto [v0, v1] = convert_int8_to_float<src_t>(src[0]);
+    return VectorizedN<float, 2>(v0, v1);
+  }
+};
 #endif
 
 #if defined(CPU_CAPABILITY_NEON)
 template <>
-struct VecConvert<float, 1, BFloat16, 1> {
-  static inline VectorizedN<float, 1> apply(
+struct VecConvert<float, 2, BFloat16, 1> {
```

Comment:
You may want to keep the `struct VecConvert<float, 1, BFloat16, 1>` too since there are cases where we still want to use half of the lane for `BFloat16` when the call range is not large enough.